### PR TITLE
Fix #12561: Synchronized coasters not synced

### LINKED ISSUE #12561: Three coasters are not synchronized correctly, just among two of them
**OS:** Windows 10
**Version:** 0.2.6
**Commit/Build:** 733bc5

I was found this issue in Three Monkeys Park scenario which is well known as that there is nice three synchronized coasters.
These coasters were not synchronized correctly. Just two of them synchronized.

Since it was fine when I tested by four coasters, I think the synchronisation does not work properly if the number of coasters are odd number(3, 5, 7 ...).

**Steps to reproduce:**
1-1. Build three coasters to synchronize
1-2. Run them
2. Or, just open Three Monkeys Park scenario

**Screenshots / Video:**
Incorrect synchronisation
All of three coasters' are synchronized.

### PR DESCRIPTION
**Pull request by tupaschoal**
As stated on issue, regression from https://github.com/tupaschoal/OpenRCT2/commit/e5e70688

I agree with @janisozaur that it's a good candidate for a replay and am willing to do so @duncanspumpkin  (and maybe document for other developers too?)

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
This wasn't in the last release so changelog entry should be removed.

**Comment by duncanspumpkin** [comment]
For the replay create as small a park as possible that exhibits the function. I can record an Replay for it and add it to the replay repo

File: distribution/changelog.txt [hunk 1]
```diff
@@ -58,6 +58,7 @@
 - Fix: [#12498] Circus construction ghost does not rotate (original bug).
 - Fix: [#12505] Stores selling multiple items can only have the first product advertised.
 - Fix: [#12506] Cannot advertise food if there are no rides in the park.
+- Fix: [#12561] Synchronized coasters not departing synced
```
Comment (tupaschoal) [inline distribution/changelog.txt:61]:
```suggestion
```

**Comment by tupaschoal** [comment]
I've removed the changelog entry. Do squash merge. Should I have incremented the network version?
I'll create the park, but I'd very much like the process of doing the replay and/or do it myself to learn.

**Comment by duncanspumpkin** [comment]
Excellent I've been getting fed up always doing the replays.

**Comment by duncanspumpkin** [comment]
Yes bump the network version. Replays should be added before merging this.

**Comment by duncanspumpkin** [comment]
Do the replays as a separate pr as a few folk are asking about this

**Pull request merged**